proxy.h

```diff
@@ -20,7 +20,9 @@
 
 #if __STDC_HOSTED__
 #include <atomic>
+#if __has_include(<format>)
```

Comment:
Why not `__cpp_lib_format`?

Reply:
Since it's a library macro, it depends on the `<format>` header